main.go

```diff
@@ -57,9 +57,9 @@ func main() {
 			EnvVar: "PLUGIN_MD5",
 		},
 		cli.StringFlag{
-			Name:   "sha265-checksum",
-			Usage:  "checksum in sha265 format",
-			EnvVar: "PLUGIN_SHA265",
+			Name:   "sha256-checksum",
+			Usage:  "checksum in sha256 format",
+			EnvVar: "PLUGIN_SHA256",
```

Comment:
Please change this to `PLUGIN_SHA256,PLUGIN_SHA265`, otherwise everybody using this typo will kill the checksum.

Reply:
done 👍